README.md

```diff
@@ -0,0 +1,19 @@
+# basysKom Toolbox
+
+## Goals
+
+- Improve the quality of the software we create through technical/organisational measures
+- Good defaults for our projects
+- Conscious decisions to make use of certain techniques at the start of the project (when there is still time)
+- Knowledge transfer within the basysKom team and with our customers
+- Provide a tools teams, developers and project managers can pick from
+
+## Non-goals
+- The goal is **NOT** to construct a heavy-weight process to be applied to all projects in a one-size-fits-all manner.
+
+## What is a tool?
+In this context, a tool is a small recipe for achieving a particular **goal**. It gives hints on how to **implement** a tool and on the potential **effort** to do so. In which situation can a tool be **usefully applied**? It outlines **caveats** and **limitations**. The idea is loosely inspired by the classic book "Design Patterns". 
```

Comment:
many years ago the tools where called recipes. we changed this at some point.
i think this is a left over.

In this context, a tool is a small guide on how to ...